readme/api/get_started/plugins.md

```diff
@@ -7,41 +7,57 @@ In this article you will learn the basic steps to build and test a plugin in Jop
-
-You will also need to have Joplin installed and running in development mode, which we'll describe later.
-
-But first install Yeoman and the Joplin Plugin Generator:
+- Install [Joplin](https://joplinapp.org/)
 
+But first install [Yeoman](https://yeoman.io/) and the [Joplin Plugin Generator](https://github.com/laurent22/joplin/tree/dev/packages/generator-joplin):
+	
 	npm install -g yo generator-joplin
 
-Then to create the plugin, run:
+Then, in the directory where you plan to develop the plugin, run:
 
 	yo joplin
 
-This will create the basic scafolding of the plugin. At the root of it, there is a number of configuration files which you normally won't need to change. Then the `src/` directory will contain your code. By default, the project uses TypeScript, but you are free to use plain JavaScript too - eventually the project is compiled to plain JS in any case.
+This will generate the basic scafolding of the plugin. At the root of it, there are a number of configuration files which you normally won't need to change. Then the `src/` directory will contain your code. By default, the project uses TypeScript, but you are free to use plain JavaScript too - eventually the project is compiled to plain JS in any case.
+
+The `src/` directory also contains a [manifest.json](https://github.com/laurent22/joplin/blob/dev/readme/api/references/plugin_manifest/) file, which contains the various information about the plugin that was set in the inital generation of the scaffolding, such as its name, homepage URL, etc. You can edit this at any time, but editing it after it has been published may cause users to have to download it again.
+
+## Setup Source Control 
+
+In your plugin directory, run: 
+
+	git init 
 
-The `src/` directory also contains a [manifest.json](https://github.com/laurent22/joplin/blob/dev/readme/api/references/plugin_manifest/) file, which you can edit to set various information about the plugin, such as its name, homepage URL, etc.
+This will setup source control.
+
+
+## Run Joplin in Development Mode
+
+You should test your plugin in Development Mode. Doing so means that Joplin will run using a different profile, so you can experiment with the plugin without risking to accidentally change or delete your data.
 
 ## Building the plugin
 
-The file `src/index.ts` already contain some basic code meant for testing the plugin. In particular it contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html), which all plugins should call to register the plugin. And an `onStart()` event handler, which will be executed by Joplin when the plugin starts.
+From the scaffolding, `src/index.ts` now contains the basic code for a Hello World plugin. 
+
+Two things to note:
+1. It contains a call to [joplin.plugins.register](https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html). All plugins call this to register the plugin in the app.
```

Comment:
Complete URL would be `https://joplinapp.org/api/references/plugin_api/classes/joplinplugins.html#register`